__tests__/app.test.js

```diff
@@ -435,4 +435,36 @@ describe('Test PATCH /api/articles/:article_id endpoint', () => {
 
 
 
+})
+
+
+describe('Test get /api/users', () => {
+  test('Test connection with /api/users', () => {
+    return request(app)
+      .get('/api/users')
+      .expect(200)
+  })
+
+  test('Test all objects returned have (username name avatar_url) keys', () => {
+    return request(app)
+      .get('/api/users')
+      .expect(200)
+      .then(({body}) => {
+        expect(body.users.length === 4).toBe(true)
```

Comment:
There is also a handy Jest matcher `.toHaveLength`
[You can read about it here](https://jestjs.io/docs/expect#tohavelengthnumber)